internal/pkg/mining/worker_test.go

```diff
@@ -698,8 +699,8 @@ func TestGenerateSetsBasicFields(t *testing.T) {
 		Clock:         th.NewFakeClock(time.Unix(1234567890, 0)),
 	})
 
-	h := types.Uint64(100)
-	w := types.Uint64(1000)
+	h := uint64(100)
```

Comment:
Note: we still need to resolve whether abi.ChainEpoch is signed or not.

Reply:
You probably already know this but FYI the uint64 vs int64 distinction doesn't matter for encoding purposes because cbor should encode both positive int64s and uint64s as major type 0.  This only becomes important for encoding if we ever want to allow a negative epoch number.

The whole specing out the semantics of algorithms in a consistent way with types that allow for intermediate negative values still applies of course.